srcpkgs/stumpwm/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'stumpwm'
+pkgname=stumpwm
+version=22.11
+revision=1
+build_style=gnu-configure
+hostmakedepends="sbcl base-devel"
+makedepends="common-lisp-ppcre common-lisp-alexandria common-lisp-clx"
+depends="common-lisp-ppcre common-lisp-alexandria common-lisp-clx"
```

Comment:
Other distros add libraries as runtime dependencies, but do anything breaks without them?

Reply:
Do you mean the libs in makedepends?